### ISSUE TITLE
X-server won't start after kernel update (crypted UEFI)

### ISSUE THREAD

**Issue by ftes** [deadhead420/arch-linux-anywhere#360]
**tldr; /etc/fstab is generated incorrectly for crypted UEFI setups. Fix provided in #359**

Debugging this error:
1. After a kernel update, the display manager doesn't start.
2. `xstart` fails: Kernel module providing graphics driver cannot be loaded (`i915` on my machine).
3. Booted kernel (`uname -r`) does not match installed version (`pacman -Q linux`).
4. Kernel on boot partition was not updated by pacman.
5. Pacman log states that new images were written to `/boot`.
6. Boot partition is not mounted at `/boot` (`umount /boot` fails).
7. `etc/fstab` has an incorrect line for boot partition. It is missing `/boot` as mount point. `cat /etc/fstab`:
```
/dev/nvme0n1p1                          vfat         ...
/dev/mapper/root        /               ext4         defaults        0       1
/dev/mapper/tmp         /tmp            tmpfs        defaults        0       0
```
8. [`configure_system.sh`](https://github.com/deadhead420/arch-linux-anywhere/blob/04dfd6eb20799d4b380890d1bedde67470fb2a5c/lib/configure_system.sh#L80) sets up `/etc/fstab` incorrectly. It uses a non-existing variable as the mount point (`$esp`, should be `$esp_mnt`).

**Comment by deadhead420** [comment]
Thanks @ftes
